packages/unified-latex-util-pegjs/grammars/latex.pegjs

```diff
@@ -172,6 +174,23 @@ square_bracket_argument
             ];
         }
 
+url
+    = escape m:"url" url:verbatim_group {
+            return [
+              createNode("macro", { content: m }),
+              url,
+            ]
+        }
+
+href
+    = escape m:"href" url:verbatim_group label:group {
+            return [
+              createNode("macro", { content: m }),
+              url,
+              label,
```

Comment:
I don't think we need label in here since the normal parsing should handle it just fine.